src/python/pants/build_graph/address_test.py

```diff
@@ -267,31 +266,6 @@ def test_address_input_from_dir() -> None:
     ).dir_to_address() == Address("a", target_name="b", generated_name="gen")
 
 
-@pytest.mark.parametrize(
```

Comment:
Because it tests a method that was removed: rather than having two implementations of spec-string rendering, `AddressInput` now records the input it was given (verbatim), so that it can render the exact value.